cwl/src/test/scala/cwl/CwlDecoderSpec.scala

```diff
@@ -21,10 +22,13 @@ class CwlDecoderSpec extends FlatSpec with Matchers {
   }
 
   it should "fail to parse broken linked cwl" in {
-      decodeCwlFile(rootPath / "brokenlinks.cwl").
-        value.
-        unsafeRunSync.
-        isLeft shouldBe true
+    val result: Either[NonEmptyList[String], Cwl] = decodeCwlFile(rootPath / "brokenlinks.cwl").
+      value.
+      unsafeRunSync
+
+    result.isLeft shouldBe true
+    "Field `run` contains undefined reference to.+wrong.cwl".r.findAllIn(result.left.get.head).size shouldBe 1
```

Comment:
uh, is there not a nicer `should contain string`?

Reply:
Not that I could see that's compatible with regex.

The message is actually
```
Field `run` contains undefined reference to `file:///var/folders/g_/4k204pzx6sg6gcnqc2bqfykxx8k880/T/c8eb79b9-eee9-4796-b374-4841ff18a27d.temp.3969925185320342713/arguments.cwl`
```
where the temporary directory changes every time